src/metrics.rs

```diff
@@ -177,6 +179,21 @@ pub fn submit(point: influxdb::Point) {
     agent.submit(point);
 }
 
+pub fn query(q: &str) -> Result<String, String> {
+    let host =
+        env::var("INFLUX_HOST").unwrap_or_else(|_| "https://metrics.solana.com:8086".to_string());
```

Comment:
dedup with https://github.com/solana-labs/solana/blob/4196cf43e84139170991c9a48c9dba2f67864801/src/metrics.rs#L40-L44 please

Reply:
Would adding these as fields to structure, and setting in new() be the correct change?

Reply:
Or, maybe not, as query() doesn't create an instance of InfluxDbMetricsWriter..

Reply:
Could just have a static helper function that fetches the values, used by both places